emu/src/cpu/arm/instructions.rs

```diff
@@ -493,9 +491,7 @@ impl From<u32> for ArmModeInstruction {
                 rm_operand_register,
                 rs_operand_register,
             }
-        } else if op_code.get_bits(25..=27) == 0b000
-            && (op_code.get_bits(4..=7) == 0b1001 || (op_code.get_bit(7) && !op_code.get_bit(4)))
-        {
+        } else if op_code.get_bits(23..=27) == 0b00000 && op_code.get_bits(4..=7) == 0b1001 {
```

Comment:
I think it's `op_code.get_bits(22..=27) == 0b000000`

<img width="328" alt="image" src="https://github.com/RIP-Comm/clementine/assets/5256712/7fa5b026-9735-4186-8645-9cd8edac4b48">

Reply:
Right! Fixed thanks